test/framework/src/main/java/org/elasticsearch/index/mapper/AbstractScriptFieldTypeTestCase.java

```diff
@@ -64,6 +65,13 @@ public abstract class AbstractScriptFieldTypeTestCase extends MapperServiceTestC
 
     protected abstract String typeName();
 
+    protected static <T extends IndexableField> void addDocument(RandomIndexWriter iw, Iterable<T> indexableFields) throws IOException {
```

Comment:
Could you stick a javadoc on this so a reader can mouse over the method and get a quick sense of what it does?

Reply:
good idea

Reply:
I'm full of good ideas. And bad ones.